exchanges/order/limits.go

```diff
@@ -130,15 +133,15 @@ func (e *ExecutionLimits) LoadLimits(levels []MinMaxLevel) error {
 				levels[x].MaxPrice)
 		}
 
-		if levels[x].MinAmount > 0 &&
-			levels[x].MaxAmount > 0 &&
-			levels[x].MinAmount > levels[x].MaxAmount {
+		if levels[x].MinimumBaseAmount > 0 &&
```

Comment:
There is no code for validating quote amount even though ByBit uses it

Reply:
At the moment conform functionality amount refers to base amounts only. This will need an update in a separate PR and below.

Reply:
Only reason I brought it up was because you are loading the quote amounts without validation in `UpdateOrderExecutionLimits`